src/actions/utxo.js

```diff
@@ -2,48 +2,84 @@
 
 import walletManager from '../crypto/wallet'
 
+import type {RawUtxo} from '../types/HistoryTransaction'
+
 import type {Dispatch} from 'redux'
 
 import type {State} from '../state'
 
+declare type ExtractReturnType = <R>((...arg: any) => R) => R;
+declare type ReturnType<Func> = $Call<ExtractReturnType, Func>;
+
 // start fetching utxo
-const _startFetching = () => ({
+const _startFetching: void => {|
+  type: 'START_FETCHING_BALANCE',
+  path: ['balance', 'isFetching'],
+  payload: null,
+  reducer: (State, boolean) => true,
+|} = () => ({
   type: 'START_FETCHING_BALANCE',
   path: ['balance', 'isFetching'],
   payload: null,
   reducer: (state, isFetching) => true,
 })
 
-const _endFetching = () => ({
+const _endFetching: void => {|
+  type: 'END_FETCHING_BALANCE',
+  path: ['balance', 'isFetching'],
+  payload: null,
+  reducer: (State, boolean) => false,
+|} = () => ({
   type: 'END_FETCHING_BALANCE',
   path: ['balance', 'isFetching'],
   payload: null,
   reducer: (state, isFetching) => false,
 })
 
-const _setUTXOs = (rawUTXOs) => ({
+const _setUTXOs: Array<RawUtxo> => {|
+  type: 'SET_UTXOS',
```

Comment:
note: we specify the type here as a constant string instead of just `string`. Flow can handle creating types out of constants and it needs this to be able to differentiate between the different types. Otherwise, if all types are just "string", Flow doesn't know what type to infer for the different possible dispatches.